libs/component-library/src/lib/progress-bar/progressBar.tsx

```diff
@@ -0,0 +1,55 @@
+import { useTheme } from '@shopify/restyle';
+import React from 'react';
+import { LayoutAnimation } from 'react-native';
+
+import { FxBox, FxBoxProps } from '../box/box';
+import { FxTheme } from '../theme/theme';
+import { WINDOW_WIDTH } from '../utils/constants';
+
+export type FxProgressBarProps = {
+  progress: number;
+  width?: number;
+  total?: number;
+  disabled?: boolean;
+} & FxBoxProps;
+
+const FxProgressBar = ({
+  width,
+  progress,
+  height = 4,
+  total = 100,
+  ...rest
+}: FxProgressBarProps) => {
+  const { spacing } = useTheme<FxTheme>();
+  const _width = width || WINDOW_WIDTH - 2 * spacing[20];
```

Comment:
Should we have it default to filling the parent component instead of `WINDOW_WIDTH - 2 * spacing[20]`?

Reply:
Could do, I based this off you will always have some margin and the standard margin was 20 horizontal. Happy to switch this off and fill the screen.

Reply:
Done